cli/index.ts

```diff
@@ -171,6 +171,26 @@ const jobPrefixOption: INamedOption<yargs.Options> = {
   }
 };
 
+const bigqueryLabelsOption: INamedOption<yargs.Options> = {
+  name: "bigquery-labels",
```

Comment:
Should this be `job-labels`? In BQ, jobs can be both in the job and in the table itself.